torch/onnx/_internal/exporter.py

```diff
@@ -633,6 +633,38 @@ def __init__(
         self._fake_context = fake_context
         self._export_exception = export_exception
 
+    def __call__(self, *args: Any, options=None, **kwargs: Any) -> Any:
+        """Runs the ONNX model using ONNX Runtime
+
+        Args:
+            args: The positional inputs to the model.
+            kwargs: The keyword inputs to the model.
+            options: The options to use for running the model with ONNX Runtime.
+
+        Returns:
+            The model output as computed by ONNX Runtime
+        """
+        import onnxruntime  # type: ignore[import]
+
+        onnx_input = self.adapt_torch_inputs_to_onnx(*args, **kwargs)
+        options = options or {}
+        providers = options.get("providers", onnxruntime.get_available_providers())
+        onnx_model = self.model_proto.SerializeToString()
+        ort_session = onnxruntime.InferenceSession(onnx_model, providers=providers)
+
+        def to_numpy(tensor):
```

Comment:
could this `to_numpy` method be replaced by just calls to `tensor.numpy(force = True)`?

Reply:
Thank you, those are good points. I will try to include the IOBinding code in this PR, but if I cannot, I will do a follow-up PR

Reply:
Tracking through https://github.com/pytorch/pytorch/issues/113687

Reply:
I think, it would also be useful to have this IObinding calling example somewhere in docs (so it can be copy-pasted into users code) or in some torch.utils function (if can be sufficiently abstracted). Testing for matching outputs between torch model and ORT-run model is a typical scenario, so would be nice to have a idiomatic way of calling ORT model with torch inputs. Maybe also this could be somehow upstreamed to ORT instead in the longer run...

Reply:
Thanks, feel free to add all comments on https://github.com/pytorch/pytorch/issues/113687 too